src/components/lyne-panel/lyne-panel.scss

```diff
@@ -0,0 +1,93 @@
+:host {
+  display: block;
+
+  --font-family: 'SBBWeb Light';
+  --letter-spacing: 0.01em;
+  --panel-width: 223;
+  --panel-height: 167;
+  --font-size: 24;
+  --line-height: 1.3;
+  --button-margin: var(--spacing-fixed-5x);
+  --panel-padding-inline: var(--spacing-responsive-s-medium);
+  --triangle-height: 33;
+
+  @include mq($from: large) {
+    --panel-width: 321;
+    --panel-height: 240.11;
+    --font-size: 32;
+    --line-height: 1.3;
+    --button-margin: var(--spacing-fixed-6x);
```

Comment:
This would be --spacing-responsive-xs-medium.

Reply:
done